src/aosm/setup.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+from codecs import open
+from setuptools import setup, find_packages
+
+try:
+    from azure_bdist_wheel import cmdclass
+except ImportError:
+    from distutils import log as logger
+
+    logger.warn("Wheel is not available, disabling bdist_wheel hook")
+
+# TODO: Confirm this is the right version number you want and it matches your
+# HISTORY.rst entry.
+VERSION = "0.2.0"
```

Comment:
Just out of curiosity, why did you directly upgrade to `0.2.0` version but skip `0.1.0` version in this initial extension version?

Reply:
Because we have sent some internal people the wheel to have a play with the tool before it gets merged.  We originally sent them 0.1.0 but made a breaking change, so updated to 0.2.0

Reply:
Only when the extension is officially released, then we need upgrade the version; otherwise we should not upgrade the version in case it confuses customers.

Reply:
As this is the first release of the extension, the version should be 1.0,0b1 per our versioning strategy.